lisa/sut_orchestrator/ready.py

```diff
@@ -66,6 +66,7 @@ def _deploy_environment(self, environment: Environment, log: Logger) -> None:
 
     def _delete_environment(self, environment: Environment, log: Logger) -> None:
         # ready platform doesn't support delete environment
+        environment.mark_clean()
```

Comment:
Add comments above to explain why need this. And as we discussed, use a switch to control it.